src/util/groupsHelpers.ts

```diff
@@ -0,0 +1,51 @@
+import { Group, Groups, Properties, PropertyOwners } from '@/types/types';
```

Comment:
Do we really want this file in the main src folder?  It will only ever be relevant to the groups slice.  Maybe put it in the `src\redux\groups? folder?

Reply:
We have types files for some of these folders, so I don't see a problem of also adding files with helper functions